freeipa/src/main/java/com/sequenceiq/freeipa/sync/FreeipaStatusInfoLogger.java

```diff
@@ -0,0 +1,53 @@
+package com.sequenceiq.freeipa.sync;
+
+import com.sequenceiq.freeipa.entity.InstanceMetaData;
+import com.sequenceiq.freeipa.entity.Stack;
+import com.sequenceiq.freeipa.entity.StackStatus;
+import com.sequenceiq.freeipa.service.stack.StackStatusService;
+import com.sequenceiq.freeipa.service.stack.instance.InstanceMetaDataService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Component
+public class FreeipaStatusInfoLogger {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FreeipaStatusInfoLogger.class);
+
+    @Inject
+    private InstanceMetaDataService instanceMetaDataService;
+
+    @Inject
+    private StackStatusService stackStatusService;
+
+    public void logFreeipaStatus(Stack stack, Set<InstanceMetaData> checkableInstances) {
```

Comment:
I think you could replace `Stack stack` with `Long stackId` as stack is only used for getting it's id

Reply:
Thanks! Resolved.